### DISCUSSION TITLE
Install not working

### DISCUSSION THREAD

**Discussion by ausittechdirect** [yolanmees/Spikster#14] [General]
Hi I would love to move over to this platform. Currently using CIPI 

Just letting you know your install does not work and also when will backups be ready?

**Comment by yolanmees** [comment]
Hi,

Thanks for trying out Spikster! What kind of error are you running into? We are still in development right now, so I’m happy to check it out!

**Comment by ausittechdirect** [comment]
Hey thanks for replying. 

It just hung at the composor install. 

Would not go past this section. 

Ubuntu 23.04
